app/src/main/java/com/github/se/cyrcle/ui/profile/EditProfileComponent.kt

```diff
@@ -52,14 +54,23 @@ fun EditProfileComponent(
   // To display the local profile picture when the user changes it, and hasn't uploaded (saved) it
   var hasTemporarilyChangedProfilePicture by remember { mutableStateOf(false) }
 
+  fun copyUser(user: User): User {
```

Comment:
Don't forget the documentation here

Reply:
There are so many ways to copy users... I just made a inner function because I used it twice and it could be a tricky bug if both copies are different. I don't think it should be give much more importance than that...

Reply:
It was just a question of organization, i am used to writing helping function outside of every other function if possible but that worked as well.